build.gradle

```diff
@@ -51,6 +51,12 @@ dependencies {
     annotationProcessor group: "io.kestra", name: "processor", version: kestraVersion
     compileOnly group: "io.kestra", name: "core", version: kestraVersion
     compileOnly group: "io.kestra", name: "script", version: kestraVersion
+
+    // JSON processing (Jackson is included with Kestra platform but ensuring availability)
+    // HTTP client and related utilities are provided by Kestra core
```

Comment:
You can remove those useless comments I think

Reply:
fixed by https://github.com/kestra-io/plugin-notion/pull/2/commits/73fb7918ba72eaf76cb93b7d35c19f2540f1f7c0